From 732ffdccc8512baaab60712c8a2992260f63e85e Mon Sep 17 00:00:00 2001
From: Vladimir Panteleev <git@thecybershadow.net>
Date: Sun, 22 Apr 2018 08:37:44 +0000
Subject: [PATCH] Print "No files to download" message as a log notice

Allow --quiet to suppress this message, which is useful for scripted
usage.
---
 src/Context.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Context.cc b/src/Context.cc
index c6603261d..dacd04c14 100644
--- a/src/Context.cc
+++ b/src/Context.cc
@@ -303,7 +303,7 @@ Context::Context(bool standalone, int argc, char** argv, const KeyVals& options)
 
   if (standalone && !op->getAsBool(PREF_ENABLE_RPC) && requestGroups.empty() &&
       !uriListParser) {
-    global::cout()->printf("%s\n", MSG_NO_FILES_TO_DOWNLOAD);
+    A2_LOG_NOTICE(MSG_NO_FILES_TO_DOWNLOAD);
   }
   else {
     if (!requestGroups.empty()) {
